queries/vimdoc/highlights.scm

```diff
@@ -24,22 +24,20 @@
   text: (_) @markup.link)
 
 (codespan
-  "`" @markup.raw.delimiter
+  "`" @markup.raw
   (#set! conceal "")
   text: (_) @markup.raw)
 
 ((codeblock) @markup.raw.block
   (#set! "priority" 90))
 
 (codeblock
-  [
-    ">"
-    (language)
-  ] @markup.raw.delimiter
+  ">" @markup.raw
+  (language) @label
   (#set! conceal ""))
```

Comment:
Missing an equivalent query for `language` here

Reply:
oh, right, because conceal will only apply to one capture.

Do multiple directives in a single pattern work yet?

Reply:
Still not. :sob: 

Would require wookayin's PR